apps/src/templates/CensusForm.jsx

```diff
@@ -41,15 +42,36 @@ const styles = {
 class CensusForm extends Component {
 
   state = {
-    value: ''
+    submission: {
+      name: '',
+      email: '',
+      role: '',
+      pledge: false,
+    }
   };
 
-  handleChange = event => {
-    this.setState({value: event.target.value});
+  handleChange = (propertyName, event) => {
+    const submission = this.state.submission;
+    submission[propertyName] = event.target.value;
+    this.setState({submission: submission});
```

Comment:
This is a nice abstraction!

nit: this line is better written as `this.setState({submission});`